epoxy-modelfactory/src/main/java/com/airbnb/epoxy/ModelProperties.kt

```diff
@@ -0,0 +1,32 @@
+package com.airbnb.epoxy
+
+import android.support.annotation.DrawableRes
+import android.view.View.OnClickListener
+import com.airbnb.paris.styles.Style
+
+interface ModelProperties {
+
+    fun getId(): String
```

Comment:
why fun over val?

Reply:
With everything else being a function it seemed inconsistent/out of place. 🤷‍♂️